types/sendsay-api/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Delete this file and make the new types fir the project-wide lint settings, thanks~

Reply:
Thanks a lot!

Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65625/commits/23d5ccaad4c839872695de028fa32d2e826dcea2